zotero2readwise/zotero.py

```diff
@@ -42,6 +42,8 @@ def __post_init__(self):
         if self.relations:
             self.relations = self.relations.get("dc:relation")
 
+        if len(self.creators) > 3:
+            self.creators = self.creators[:3] + ["et al."]
         self.creators = ", ".join(self.creators) if self.creators else None
```

Comment:
@noeleont After joining `self.creators` and in case it's not None, check the number of characters and trim if character counts are greater than Readwise limits.
So, something like the following?

```python
self.creators = ", ".join(self.creators) if self.creators else None
if self.creators and (len(self.creators) >= 1023):
    self.creators = self.creators[:1015] + ", et al."
```

Reply:
@e-alizadeh I updated the code to reflect your idea while keeping only whole author names, what do you think?